systemd build broken

https://oss-fuzz-build-logs.storage.googleapis.com/index.html

```
Step #4: [185/277] Compiling C object 'src/basic/basic@sta/mount-util.c.o'.
Step #4: FAILED: src/basic/basic@sta/mount-util.c.o 
Step #4: clang  -Isrc/basic/basic@sta -Isrc/basic -I../../src/systemd/src/basic -I. -I../../src/systemd/ -Isrc/libsystemd-network -I../../src/systemd/src/libsystemd-network -I../../src/systemd/src/libsystemd/sd-network -I../../src/systemd/src/libsystemd/sd-netlink -I../../src/systemd/src/libsystemd/sd-id128 -I../../src/systemd/src/libsystemd/sd-hwdb -I../../src/systemd/src/libsystemd/sd-device -I../../src/systemd/src/libsystemd/sd-bus -Isrc/core -I../../src/systemd/src/core -Isrc/libudev -I../../src/systemd/src/libudev -Isrc/udev -I../../src/systemd/src/udev -Isrc/login -I../../src/systemd/src/login -Isrc/timesync -I../../src/systemd/src/timesync -Isrc/resolve -I../../src/systemd/src/resolve -Isrc/journal -I../../src/systemd/src/journal -Isrc/systemd -I../../src/systemd/src/systemd -Isrc/shared -I../../src/systemd/src/shared -Xclang -fcolor-diagnostics -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -O0 -g -Wextra -Werror=undef -Wmissing-include-dirs -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Werror=missing-prototypes -Werror=implicit-function-declaration -Werror=missing-declarations -Werror=return-type -Werror=incompatible-pointer-types -Werror=format=2 -Wstrict-prototypes -Wredundant-decls -Wmissing-noreturn -Wshadow -Wendif-labels -Wstrict-aliasing=2 -Wwrite-strings -Werror=overflow -Wdate-time -Wnested-externs -ffast-math -fno-common -fdiagnostics-show-option -fno-strict-aliasing -fvisibility=hidden -fstack-protector -fstack-protector-strong --param=ssp-buffer-size=4 -Wno-unused-parameter -Wno-missing-field-initializers -Wno-unused-result -Wno-error=nonnull -Werror=shadow -Wno-typedef-redefinition -Wno-gnu-variable-sized-type-not-at-end -include config.h -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -fPIC -pthread -fvisibility=default -MMD -MQ 'src/basic/basic@sta/mount-util.c.o' -MF 'src/basic/basic@sta/mount-util.c.o.d' -o 'src/basic/basic@sta/mount-util.c.o' -c ../../src/systemd/src/basic/mount-util.c
Step #4: ../../src/systemd/src/basic/mount-util.c:31:10: fatal error: 'libmount.h' file not found
Step #4: #include <libmount.h>
Step #4:          ^~~~~~~~~~~~
Step #4: 1 error generated.
Step #4: [186/277] Compiling C object 'src/basic/basic@sta/parse-util.c.o'.
```
@titanous @keszybz - can you please take a look

@titanous @keszybz - i still see error locally when i install libmount-dev through DockerFile, please submit a pull request with fix and verify using
python infra/helper.py build_fuzzers systemd

```
../../src/systemd/src/basic/mount-util.c:31:10: fatal error: 'libmount.h' file not found
#include <libmount.h>
```

--- a/projects/systemd/Dockerfile
+++ b/projects/systemd/Dockerfile
@@ -16,7 +16,8 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER jonathan@titanous.com
-RUN apt-get update && apt-get install -y gperf m4 gettext libcap-dev python3-pip &&\
+RUN apt-get update &&\
+    apt-get install -y gperf m4 gettext libcap-dev python3-pip libmount-dev pkg-config &&\
     pip3 install meson ninja
 RUN git clone --depth 1 https://github.com/systemd/systemd systemd
 WORKDIR systemd

